aspnet/tutorials/web-api-help-pages-using-swagger/sample/src/TodoApi/wwwroot/swagger/ui/index.html

```diff
@@ -32,19 +35,17 @@
 
   <script type="text/javascript">
     $(function () {
-      var url = window.location.search.match(/url=([^&]+)/);
-      if (url && url.length > 1) {
-        url = decodeURIComponent(url[1]);
-      } else {
-        url = "/swagger/v1/swagger.json";
-      }
```

Comment:
Allowing the discovery url to be provided in the URL introduces a potential phishing vulnerability and really isn't necessary for most (if not all) Swashbuckle use-cases. So, best to just remove that code